tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "strict": true,
-    "target": "es6",
+    "target": "ESNext",
```

Comment:
I recall some tooling, I think it was Cypress, that has trouble consuming modern JavaScript. They require third-parties to be CommonJS. @thepassle, should that be our concern?

Reply:
This sounds a little odd to me. A browser launcher expects code to be cjs? 🫠